storageClient/api.go

```diff
@@ -47,6 +47,32 @@ type Group struct {
 	EncryptedGroupKey string    `json:"encrypted_group_key"`
 }
 
+// GroupAccessKeyRequest wraps the values to get or create an access key
+type GroupAccessKeyRequest struct {
+	GroupID            uuid.UUID `json:"group_id"`
+	RecordType         string    `json:"record_type"`
+	RecordID           string    `json:"record_id"`
+	EncryptedAccessKey string    `json:"encrypted_access_key"`
+	PublicKey          string    `json:"public_key"`
+}
+
+// GroupRecord wraps the values of a record for a group
+type GroupRecord struct {
```

Comment:
When is this returned? I hesitate to have a specific per-record struct like this, as opposed to a wrapping struct like when searching records. I worry this may cause type problems down the line.

Reply:
After you share a record with a group

Reply:
Yup, and based on our talk this should change to using pds.ListedRecord and expanding the GetEAKResponse to include a slice of wrapping keys.